src/debug/migrate-user-metadata-1.ts

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
you are right. I did that because initially the first PR had kratos involved to store the informations. now hat we don't store anything on kratos anymore I can actually have a migration only script.